lib/features/login/presentation/widgets/home_page_buttons.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+// This uses the IconButton class seen here https://api.flutter.dev/flutter/material/IconButton-class.html
+
+class HomePageButtons extends StatelessWidget {
```

Comment:
Since this class handles only 1 button at a time let's rename it to:
```suggestion
class HomePageButton extends StatelessWidget {
```